src/ocamlorg_frontend/url.ml

```diff
@@ -35,6 +35,7 @@ let releases = "/releases"
 let release v = "/releases/" ^ v
 let workshop v = "/workshops/" ^ v
 let blog = "/blog"
+let atom = "/atom"
```

Comment:
Perhaps a more standard URL for RSS feed is `/feed.xml`. But also if you generate the file in `asset`, you don't need to add a route for it. 
```suggestion
```